helix-rest/src/test/java/org/apache/helix/rest/client/TestCustomRestClient.java

```diff
@@ -161,4 +179,35 @@ protected JsonNode getJsonObject(HttpResponse httpResponse) throws IOException {
       return new ObjectMapper().readTree(_jsonResponse);
     }
   }
+
+  /**
+   * This MockCustomRestClientFactory is necessary to have for testing because once an INSTANCE is
+   * initialized in CustomRestClientFactory while running "mvn test" , it will no re-initialize the
```

Comment:
Please fix typos.

Reply:
Removed. Thanks. I realize this class can be removed and not needed.